### PR TITLE
80 uninvite customer user

### PR DESCRIPTION
**Pull request by allynt**

### CODE REVIEW

File: astrosat_users/models/models_customers.py
Comment (marksmall) [inline astrosat_users/models/models_customers.py:203]: Previous uses of `context` I've seen have been like:

```python
context["user"] = user
context["customer"] = customer
```
Is this just a different way of doing the same thing? `update` suggests you are updating the user that already exists in the `context`, why not just overrwrite it?

File: astrosat_users/templates/astrosat_users/email/invitation_message.txt
Comment (marksmall) [inline astrosat_users/templates/astrosat_users/email/invitation_message.txt:30]: I don't see the `user` being used in this template, possibly in `astrosat_users/email/message_base.txt` though. Does `user` need to be added/updated in the `context`?

File: astrosat_users/templates/astrosat_users/email/uninvitation_message.txt
Comment (marksmall) [inline astrosat_users/templates/astrosat_users/email/uninvitation_message.txt:15]: or here

**Review by marksmall — COMMENTED** [review]
Couple of questions on `uninvite` context, but generally looks good. Will approve if you can answer those questions.

File: astrosat_users/templates/astrosat_users/email/invitation_message.txt
Comment (allynt) [reply astrosat_users/templates/astrosat_users/email/invitation_message.txt:30]: No, I suppose not.

File: astrosat_users/models/models_customers.py
Comment (allynt) [reply astrosat_users/models/models_customers.py:203]: They do the same thing.  I don't think one is any better than another.  In general, if I am merging two dictionaries I will use `update` otherwise I will use explicit keys as you suggest.

**Review by marksmall — APPROVED** [review]
You've answered my questions. I'll leave it up to you to decide if you wanna remove the user from the context, since it isn't used, just not a barrier to merging this code.

File: astrosat_users/models/models_customers.py
Comment (allynt) [reply astrosat_users/models/models_customers.py]: Your suggestion is a good one - using `[]` is much more efficient than using `update`.  I have done this in 86d2de2ab8072c6e996211a1a3947d0e068c5671.

**Pull request merged**
